### ISSUE TITLE
PHP Warning:  proc_open() expects parameter 1 to be string

### ISSUE THREAD

**Issue by johnbburg** [overtrue/phplint#61]
I am running int this warning that appears to halt execution. Running PHP 7.1.26. I'm using the yml file contents from the README, only with a modification to point to my project path.

After running into the error the first time, I did modify the code to var_dump() the contents of `$commandline`, which is the referenced argument in the error message.

```
$ ./vendor/bin/phplint -vvv
phplint 1.0.2 by overtrue and contributors.

Loaded config from "/vagrant/.phplint.yml"

Options: {"jobs":10,"exclude":["vendor"],"extensions":["php","module","inc"],"path":".\/public","configuration":"\/vagrant\/.phplint.yml","verbose":true}

/vagrant/vendor/symfony/process/Process.php:294:
array(5) {
  [0] =>
  string(12) "/usr/bin/php"
  [1] =>
  string(24) "-d error_reporting=E_ALL"
  [2] =>
  string(20) "-d display_errors=On"
  [3] =>
  string(2) "-l"
  [4] =>
  string(54) "/vagrant/public/includes/filetransfer/filetransfer.inc"
}
PHP Warning:  proc_open() expects parameter 1 to be string, array given in /vagrant/vendor/symfony/process/Process.php on line 296
PHP Stack trace:
PHP   1. {main}() /vagrant/vendor/overtrue/phplint/bin/phplint:0
PHP   2. Overtrue\PHPLint\Console\Application->run() /vagrant/vendor/overtrue/phplint/bin/phplint:32
PHP   3. Overtrue\PHPLint\Console\Application->doRun() /vagrant/vendor/symfony/console/Application.php:122
PHP   4. Overtrue\PHPLint\Console\Application->doRunCommand() /vagrant/vendor/symfony/console/Application.php:215
PHP   5. Overtrue\PHPLint\Command\LintCommand->run() /vagrant/vendor/symfony/console/Application.php:858
PHP   6. Overtrue\PHPLint\Command\LintCommand->execute() /vagrant/vendor/symfony/console/Command/Command.php:240
PHP   7. Overtrue\PHPLint\Command\LintCommand->executeLint() /vagrant/vendor/overtrue/phplint/src/Command/LintCommand.php:191
PHP   8. Overtrue\PHPLint\Linter->lint() /vagrant/vendor/overtrue/phplint/src/Command/LintCommand.php:280
PHP   9. Overtrue\PHPLint\Process\Lint->start() /vagrant/vendor/overtrue/phplint/src/Linter.php:106
PHP  10. proc_open() /vagrant/vendor/symfony/process/Process.php:296

                                                          
  [Symfony\Component\Process\Exception\RuntimeException]  
  Unable to launch a new process.                         
                                                          

Exception trace:
 () at /vagrant/vendor/symfony/process/Process.php:299
 Symfony\Component\Process\Process->start() at /vagrant/vendor/overtrue/phplint/src/Linter.php:106
 Overtrue\PHPLint\Linter->lint() at /vagrant/vendor/overtrue/phplint/src/Command/LintCommand.php:280
 Overtrue\PHPLint\Command\LintCommand->executeLint() at /vagrant/vendor/overtrue/phplint/src/Command/LintCommand.php:191
 Overtrue\PHPLint\Command\LintCommand->execute() at /vagrant/vendor/symfony/console/Command/Command.php:240
 Symfony\Component\Console\Command\Command->run() at /vagrant/vendor/symfony/console/Application.php:858
 Symfony\Component\Console\Application->doRunCommand() at /vagrant/vendor/symfony/console/Application.php:215
 Symfony\Component\Console\Application->doRun() at /vagrant/vendor/symfony/console/Application.php:122
 Symfony\Component\Console\Application->run() at /vagrant/vendor/overtrue/phplint/bin/phplint:32

phplint [--exclude EXCLUDE] [--extensions EXTENSIONS] [-j|--jobs JOBS] [-c|--configuration CONFIGURATION] [--no-configuration] [--no-cache] [--cache CACHE] [--no-progress] [--json [JSON]] [--] [<path>]...
```

Any help would be appreciated.

**Comment by bastianschwarz** [comment]
Have you checked what symfony versions are installed? I ran into this issue when using composer --prefer-lowest in our CI

My best guess without digging in further is a possible symfony version mismatch:

```
  - Installing symfony/process (v3.0.0): Loading from cache
  - Installing symfony/finder (v3.0.0): Loading from cache
  - Installing symfony/debug (v2.8.0): Loading from cache
  - Installing symfony/console (v3.2.0): Loading from cache
  - Installing overtrue/phplint (1.2.0): Loading from cache
```

Try setting a minimal version for symfony/process since the command line building was extended in later versions. Not ideal, but I could work as a quick workaround.

Ideally, the minimal version in this package could be set to 3.4 (or whatever symfony version this was fixed in) since the current constraints might break.
